scout/server/blueprints/diagnoses/views.py

```diff
@@ -39,6 +39,8 @@ def count_diagnoses():
 def api_diagnoses():
     """Return JSON data about diseases in the database."""
     query: Union[str, None] = request.args.get("query") or None
+    if query is None or len(query) < 3:
```

Comment:
👍

Reply:
Looks good, but I'm afraid you would have to make a couple of accompanying changes to the UI, where this is used? Like either catching that error code, or ensuring the user can't actually submit < 3 chars?
These hang indefinitely without explanation with fewer than three chars. 😊 
![Screenshot 2024-05-03 at 09 15 20](https://github.com/Clinical-Genomics/scout/assets/758570/9d383c6a-3e45-4b7d-acc6-03bd357babde)

Reply:
Right. I was thinking at [another recent change](https://github.com/Clinical-Genomics/scout/pull/4592) to the functionality. So now if you search for OMI you have 3 chars but it's a useless query anyway. Why don't we simplify and just keep it so that the user can insert anything, even blank string. I don't think it will hang on stage or prod because of the performance of mongo there. And if it hangs they'll look for something else anyway. It was like this even before..

Reply:
Right, and they can easily ask for all diagnoses using the regular endpoint anyway, so I guess it's not that much of a difference if they can do it via the API. We can modify after the first DDOS... 😁